apps/src/lib/levelbuilder/code-docs-editor/ProgrammingExpressionsTable.jsx

```diff
@@ -103,16 +108,26 @@ export default function ProgrammingExpressionsTable({
     }).then(response => {
       if (response.ok) {
         setItemToDelete(null);
-        fetchExpressions(selectedEnvironment, selectedCategory, currentPage);
+        fetchExpressions(
```

Comment:
can we change this to `fetchResults` if it's not too much trouble?

Reply:
yes! done:)